README.md

```diff
@@ -30,3 +30,23 @@ bash build-profiling-ffi.sh /opt/libdatadog
 #### Build Dependencies
 
 Rust 1.69 or newer with cargo. Some platforms may need protoc; others have it shipped in prost-build.
+
```

Comment:
Beyond the scope of this PR, but just a thought...should we offer a dev docker image for working on datadog with all the necessary tools pre-installed? Some client libraries do this to make setting up a dev env easier.